.travis.yml

```diff
@@ -12,12 +12,22 @@ addons:
     - liblapack-dev
 
 env:
-  - FC=gfortran
+  - FC=gfortran OCCA_HEAD=HEAD
+  - FC=gfortran OCCA_HEAD=527494c
```

Comment:
Surely the second line overwrites the first.

Reply:
It is like the compiler specification; the build matrix is a combination of the different lines in runtime, environment, and exclusions/inclusions. So we get 2 compiler options x 2 env options = 4 runs

https://docs.travis-ci.com/user/customizing-the-build/#build-matrix